apps/console/src/features/server-configurations/components/governance-connectors/dynamic-governance-connector.tsx

```diff
@@ -22,7 +22,9 @@ import { Message, Text } from "@wso2is/react-components";
 import camelCase from "lodash-es/camelCase";
 import get from "lodash-es/get";
 import kebabCase from "lodash-es/kebabCase";
-import React, { FunctionComponent, ReactElement, ReactNode, useEffect, useState } from "react";
+import { IdentityAppsApiException } from "modules/core/dist/types/exceptions/identity-apps-api-exception";
```

Comment:
Shall we add the relative path?

Reply:
+1

Reply:
Resolved by https://github.com/wso2/identity-apps/pull/3982/commits/8fd831d8584b09497fbab6218a7b567d712d87df